### PR TITLE
OP-347 Cumulative script update for 1.12

### PR DESCRIPTION
**Pull request by mizzioisf**
Batch of updates for the incoming 1.12 release.

Last update: clarified database server console messages
Update: added check for SERVER feature

New features:
- Full 64bit support on Windows, also for DICOM ! (no more ugly DICOM workarounds - see https://openhospital.atlassian.net/browse/OP-706)
- SERVER mode support (for portable installations) -> give it a try !
- Arabic Language support - oh.sh -l ar
- JRE 11 !
- MariaDB 10.6.x !

Along with the usual bug fixes, cosmetic/syntax changes, etc.

### CODE REVIEW

File: oh.ps1 [hunk 1]
```diff
@@ -195,15 +196,18 @@ else {
...
-	$script:JAVA_PACKAGE_ARCH="i686";
-	#$script:MYSQL_ARCH=32;
-}
+#
+# NOT NEEDED ANYMORE FROM OH 1.12-dev !
+#
 
-######## MySQL/MariaDB Software
-# MariaDB
+#if ( $DICOM_ENABLE -eq "on" ) {
```
Comment (dbmalkovsky) [inline oh.ps1:203]: Is this still needed?  I thought when upgraded/updated the DICOM jars and native libs this wasn't needed any longer.

Comment (dbmalkovsky) [reply oh.ps1:203]: ugh ... I didn't notice the comment characters at the beginning of the line 😦

File: oh.ps1 [hunk 2]
```diff
@@ -523,6 +534,12 @@ function initialize_database {
 }
 
 function start_database {
+	Write-Host "Checking if MySQL server is running..."
+	if ( ( Test-Path "$OH_PATH/$TMP_DIR/mysql.sock" ) -or ( Test-Path "$OH_PATH/$TMP_DIR/mysql.pid" ) ) {
+		Write-Host "MySQL server already running! Exiting."
```
Comment (dbmalkovsky) [inline oh.ps1:539]: There seems to be a move away from saying "MySQL" and using "database" instead which I think is a good idea given MySQL and MariaDB;  so there are other places that should change too?  Agreed?

Comment (mwithi) [reply oh.ps1]: A second pass will be done for the doc

**Pull request merged**